ops/test-capabilities.sh

```diff
@@ -4,6 +4,6 @@ docker run -it \
     --network=suborbital_scn \
     -v "${PWD}/examples/http-cap:/tmp" \
     -e SAT_HTTP_PORT=8079 \
-    -e SAT_CONTROL_PLANE=scc-control-plane:8081 \
+    -e SAT_CONTROL_PLANE=scn-control-plane:8081 \
```

Comment:
```suggestion
    -e SAT_CONTROL_PLANE=scn-controlplane:8081 \
```

The images will be renamed to the above, but they don't exist just yet